src/sentry/middleware/proxy.py

```diff
@@ -11,7 +11,8 @@ def process_request(self, request):
             # HTTP_X_FORWARDED_FOR can be a comma-separated list of IPs.
             # Take just the first one.
             real_ip = real_ip.split(",")[0]
-            if ':' in real_ip:
+            if '.' in real_ip and ':' in real_ip:
```

Comment:
Can we switch these around?

To `if ':' in real_ip and '.' in real_ip:'` since it'll be a tad faster for the common case where `:` doesn't exist.